src/redux/books/apiBook.js

```diff
@@ -0,0 +1,85 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { ADD_ARR_BOOK, ADD_BOOK, REMOVE_BOOK } from './books';
+
+const UNIQ_ID = 'IUmpICCqFqLeGrbCEOb9';
+const BASE_URL = `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${UNIQ_ID}/books/`;
+
+export const getBooks = createAsyncThunk(ADD_ARR_BOOK, async () => {
+  const response = await fetch(BASE_URL);
+  const data = await response.json();
+  // console.log('asynk');
+  // console.log(data.item1);
+  // console.log(data);
```

Comment:
- [ ] Kindly make sure that you remove all commented-out/ unwanted pieces of code from your code base. It makes it look messy and cluttered.